space.coin.wallet.json

```diff
@@ -0,0 +1,34 @@
+{
+  "id" : "space.coin.wallet",
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version" : "24.08",
+  "sdk" : "org.freedesktop.Sdk",
+  "base" : "org.electronjs.Electron2.BaseApp",
+  "base-version" : "24.08",
+  "base-commit" : "737af19c7c24b08befe8580439437efea9ca9163b8a1ed94b8be78d3d46b39b8",
+  "command" : "electron-wrapper",
+  "finish-args" : [
+    "--socket=x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--device=dri",
+    "--socket=pulseaudio",
+    "--filesystem=home",
+    "--env=TMPDIR=/var/tmp",
+    "--share=network",
+    "--talk-name=org.freedesktop.Notifications",
+    "--device=all"
+  ],
+  "modules" : [
```

Comment:
where is the application?

Reply:
The application is located in GitHub releases https://github.com/CoinSpace/CoinSpace/releases/tag/v6.8.0

Reply:
https://github.com/CoinSpace/CoinSpace/releases/download/v6.8.0/space.coin.wallet.flatpak

Reply:
so how is the buildbot gonna guess this?

Reply:
The application is built via GitHub Actions https://github.com/CoinSpace/CoinSpace/actions

How do I specify this in the manifest?